buildpipeline/perf-pipeline.groovy

```diff
@@ -39,7 +27,7 @@ def windowsBuild(String arch, String config, String pgo, boolean isBaseline) {
     stash name: "nt-${arch}-${pgo}${baselineString}-test-artifacts", includes: 'bin/tests/**'
 }
 
-def windowsPerf(String arch, String config, String uploadString, String runType, String opt_level, String jit, String pgo, String scenario, boolean isBaseline, boolean isProfileOn, String testFolder) {
+def windowsPerf(String arch, String config, String uploadString, String runType, String opt_level, String jit, String pgo, String scenario, boolean isBaseline, boolean isProfileOn, int slice) {
```

Comment:
Maybe we can make this an enum? That would allow us to use the integer indexing into the JSON file, but still make the slices more human readable then slice == 4?

Reply:
I see what you're saying, but the whole point of this was to keep the knowledge out of the groovy file. If we do what you're suggesting, then we'd have to have additional files updated when we add new perf suites. Doing it this way was @brianrob's idea.

Reply:
Yup, I did ask for an integer so that we can de-couple the number of slices from what runs in them.

@DrewScoggins what would you want the enum to look like?